### PR TITLE
Use configuration for payment provider information

### PR DESCRIPTION
**Pull request by gldraphael**
Once this has been merged, the following script must be run:

```sql
UPDATE Registrations
    SET PaymentMethod = PaymentMethodId
    WHERE PaymentMethod is null
GO

UPDATE Orders
    SET PaymentMethod = PaymentMethodId
    WHERE PaymentMethod is null
GO
```

Following which the `PaymentMethodId` fields can be removed from the `Registrations` and `Orders` tables.

### CODE REVIEW

File: src/EventManagement.Services/PaymentMethodService.cs [hunk 1]
```diff
@@ -4,40 +4,37 @@
...
 using losol.EventManagement.Infrastructure;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
 
 namespace losol.EventManagement.Services
 {
 	public class PaymentMethodService : IPaymentMethodService
 	{
-		private readonly ApplicationDbContext _db;
+        private readonly IEnumerable<PaymentMethod> paymentMethods;
```
Comment (losviko) [inline src/EventManagement.Services/PaymentMethodService.cs]: Could/should we move this to the OrdersService?

File: src/EventManagement.Web/Startup.cs [hunk 2]
```diff
@@ -213,18 +196,8 @@ public virtual void Configure(IApplicationBuilder app, IHostingEnvironment env,
             }
 
             app.UseStaticFiles();
-
             app.UseAuthentication();
 
-            // TODO reenable
```
Comment (losviko) [inline src/EventManagement.Web/Startup.cs]: Please keep this....

File: src/EventManagement.Web/Startup.cs [hunk 3]
```diff
@@ -57,18 +55,6 @@ public virtual void ConfigureServices(IServiceCollection services)
                 .AddDefaultTokenProviders()
                 .AddMagicLinkTokenProvider();
 
-            // Require SSL
```
Comment (losviko) [inline src/EventManagement.Web/Startup.cs]: Please keep this, will be reenabled.

File: src/EventManagement.Services/PaymentMethodService.cs (hunk 1 above)
Comment (gldraphael) [reply src/EventManagement.Services/PaymentMethodService.cs]: What makes you think so?

**Pull request merged**